components/automate-chef-io/content/docs/ldap.md

```diff
@@ -25,9 +25,9 @@ Once the user has provided a username and password at the login screen, Chef
 Automate goes through a sequence of operations to complete the login: Connect,
 Bind, User Search, Login Bind, and Group Search.
 
-### Authorization with LDAP
+#### Authorization with LDAP
 
-Chef Automate supports using your LDAP server to authenticate users and define their permissions based on [LDAP group memberships]({{< ref "authorization-overview.md" >}}).
+Chef Automate supports defining permissions for [LDAP users and their groups]({{< ref "authorization-overview.md" >}}).
```

Comment:
is there an anchor link we can link to instead? it looks like no, but thought i would ask.

Reply:
I'll look into it. Maybe a brief extra paragraph wouldn't hurt.... 🤔

Reply:
"Subjects" makes sense to link to, I think. Changed it accordingly.